### PR TITLE
Initial opendata.swiss profile

### PR DESCRIPTION
**Pull request by tpluscode**
I added a basic profile for opendata.swiss metadata to its own profile.

Need to add more tests and in barnard the support for `code:imports`. Otherwise this is functional already and has been battle-tested on FOAG milk cubes

### CODE REVIEW

File: test/shorten-report.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const SHACL_PLAYGROUND_URL = 'https://shacl-playground.zazuko.com/'
+const SHORTENER = 'https://s.zazuko.com/api/v1/shorten/'
+
+;(async function () {
```
[the remark this answers is not in the document]
Comment (tpluscode) [reply test/shorten-report.js:6]: Well, you need a semicolon somewhere. Here, or at the end of the line above

**Comment by tpluscode** [comment]
Thank you @Rdataflow. This is still WIP as we are exploring options for two opendata.swiss profiles, actually. One would be specifically targeting the handbook requirements, and the other specific to Lindas because the CKAN harvester actually does some reprocessing. Thus, we need to take into account that a cube on Lindas does not 100% match the opendata.swiss requirements

**Comment by kronmar** [comment]
@tpluscode , @Rdataflow I had a first look and here are some comments with issues our cubes had (I tested with [this one](https://environment.ld.admin.ch/foen/nfi/nfi_C-1266/cube/2023-3) but shouldn't really mather)

- `dct:creator`: violation because it must have the same value as `schema:creator`. Currently on Prod, only seven cubes have a `schema:creator`. So all cubes would have to add it.
- `dct:publisher`: violation, value needs to be a `foaf:Organization`. Currently, all cubes use a string. So all cubes would have to change that. One could reuse the [creator ](https://register.ld.admin.ch/opendataswiss/org/bundesamt-fur-umwelt-bafu), but the URI is not (yet) defined as a `foaf:Organization` - but that's a quick thing I would assume.
- `dcat:contactPoint` and `schema:contactPoint`: violation, they need to be the same. Many cubes on PROD use both and most of the time as a blank node. Can two blank nodes ever be the same? They could have the same meaning, but they'll use different places in the memory, right? Thus, even if one copy-pastes the contact points, validation still fails for me - I assume because the blank nodes are not the "same" in this strict sense. Furthermore, `dcat:contactPoint` asks for `a vcard:Organization`, using `vcard:hasEmail ?mail ; vcard:fn ?name`. Most cubes on PROD use then for `schema:contactPoint` with `schema:email ?mail ; schema:name ?name`. So most cubes use a completely different structure. What should be same, is those two properties I would say. Not the nodes.
- `dcat:theme`: Violation again. [The cubes on PROD](https://s.zazuko.com/3t2K2Km) use a different set of themes. Really not sure, what to do here, and from where the themes on LINDAS come from.
- `dcat:keyword`: Violation, but this time I know why, I just messed up, and didn't add a language tag to one of the keywords, so my bad. 

Note: it may sound, that I am against changing all cubes on PROD or something like that. I'm not, if there is a good reason. And for example in the first case, I can see that there is a reason and I would say for consistency's sake it's a good one.

**Comment by tpluscode** [comment]
I pushed a solid update to this branch. 

First of all, I moved some properties to a dedicated `opendataswiss-lindas` profile. This is to match the expectation of the [CKAN plugin](https://github.com/zazuko/trifid/tree/main/packages/ckan) for "publishable" cubes.

This leaves the opendata.swiss profile more accurately matching the handbook. For discussion, please follow https://gitlab.ldbar.ch/bafu/visualize/-/issues/609. tl;dr; I see two options:

1. require cubes on Lindas to explicitly pass the profile validation if it is intended for publishing on ODS
2. keep some logic in the Lindas `/ckan` endpoint provided by the aforementioned plugin. It already does some preprocessing but ideally it would be kept to a minimum 

We can find middle ground but in any case, at least latest published versions of cubes may require patching on Lindas or publishing new versions

**Comment by tpluscode** [comment]
Included in #141

**Pull request closed without merging**